aries-site/src/pages/templates/user-feedback-collection.mdx

```diff
@@ -0,0 +1,103 @@
+
+### When not to use
+Don’t collect feedback in the following use cases:
+- If it disrupts a user flow or is intrusive in any way.
+- There is no actionable plan for what to do with the data.
+- In locations where users do not have enough context to form a valid opinion. 
+- There are too many instances soliciting feedback which can [dilute feedback effectiveness](#sampling-frequency).
+
+### Sampling frequency
+Avoid over sampling and/or diluting effectiveness by limiting who is asked for feedback and 
+how often. Consider implementing "rules of engagement" specifying under what conditions an 
+individual user should be solicited for feedback. Example engagement rules might be:
+- Has not submitted response to this questionare in the last 90 days.
+- Has not been asked to complete this questionare in the last 30 days.
+- Has not been asked to complete any questionare in the last 7 days.
+
+## Anatomy
+<Example plain>
+  <FeedbackAnatomy />
+</Example>
+
+| Label  | Region       | Purpose                                                              |                  Required                   | Notes                                                                                                      |
+| ------ | ------------ | -------------------------------------------------------------------- | :-----------------------------------------: | ---------------------------------------------------------------------------------------------------------- |
+| **1**  | **Title**    | Title prompting user for their feedback.                             | <Checkmark a11yTitle="true" size="small" /> | <span>Use the [semantically correct heading level](/foundation/typography?q=semantically#semantic-usage-of-heading-levels) for the title. "We'd love your feedback" is a nice, appreciative default title.</span>       |
+| **2**  | **Body**     | The body may be populated as many questions as appropriate.          | <Checkmark a11yTitle="true" size="small" /> | <span>Question types can include [StarRating](https://v2.grommet.io/starrating), [ThumbsRating](https://v2.grommet.io/thumbsrating), select all that apply (e.g. checkbox group), choose one (e.g. RadioButtonGroup), open ended (e.g. TextArea), or any other input types.</span> |                                                             
+| **3**  | **Footer **  | Contains two actions: "Submit" and "Dismiss" if used in Layer modal. Otherwise only "Submit" is needed inline. | <Checkmark a11yTitle="true" size="small" /> |  Submit button sends survey responses to the HPE customer experience team and initiates thank you message. If presented in a modal, the Dismiss button allows the user to bypass the survey and continue their product experience. |
+
+## Feedback setup and submission
+To iniate user feedback collection, please contact the <a href={`mailto:${contact.email}?subject=Collecting%20user%20feedback%20in%20a%20product%20experience`}>unified customer experience team</a> for the following steps:
```

Comment:
I'm not sure the wording of "for the following steps" makes sense to me. Will Doris's team send the following to the person who contacts? Or it's more that they will contact Doris to initiate the interaction. Then, the following back-and-forth will occur with Doris to set up the feedback

Reply:
so they will just work with Doris for this information.. Doris will give them everything they need for the API calls

Reply:
I can take a stab at re-writing for clarity.